### DISCUSSION TITLE
是否需要将参数改为url参数

### DISCUSSION THREAD

**Discussion by ShilongLee** [ShilongLee/Crawler#50] [Ideas]
最近，我经常收到一些大家邮件和issue，发现大家对该项目的使用似乎不是那么轻松。
> 就比如，当大家想要获取抖音的视频详情时，api是这样的`http://localhost:8080/douyin/detail?id=123`
> 但实际上大多数情况下很多开发者更喜欢使用路径参数`http://localhost:8080/douyin/detail/123`，这样的写法对于使用者来说更简洁方便，也更利于用浏览器手动输入url访问。
> 对于评论或者回复等参数较多的接口可以只把id放在url路径参数中`http://localhost:8080/douyin/comment/123?offset=10&limit=5`。

大家更喜欢哪种呢，欢迎讨论👏
如果您支持更改为路径参数，请评论表情👍，否则评论👎，其他表情视为中立态度。

**Comment by maryjeck** [comment]
![image](https://github.com/user-attachments/assets/67082950-172e-421d-85aa-086f953036d4)

现在就很方便啊

**Comment by luoysuan** [comment]
感觉都差不多，统一好风格就好了

**Comment by demodeom** [comment]
http://localhost:8080/douyin/detail?id=123 个人更喜欢 这种风格， 路径就是路径， 参数就是参数，特别是在 api 应用场景下，这种风格更清晰，一眼看过去一清二楚
